examples/mnist/Instructions.md

```diff
@@ -0,0 +1,27 @@
+<p align="center"><b>Setup, build and run tiny-ddn on various OS</b><p align="center">
```

Comment:
`various OS` > `various operating systems` OS should be plural after `various` adjective.